uilib/lib/icons/BracketLeft.d.ts

```diff
@@ -0,0 +1,3 @@
+import React from 'react';
+declare const BracketLeft: (props: any) => React.ReactElement;
+export default BracketLeft;
```

Comment:
If I'm right, `react-native-svg` library exposes props type for his SVG component.

```suggestion
import React from 'react';
import type {  SvgProps  } from 'react-native-svg';

declare const BracketLeft: (props: SvgProps) => React.ReactElement;
export default BracketLeft;
```